altinvoice.php

```diff
@@ -55,12 +55,17 @@ function altinvoice_civicrm_alterMailParams(&$params, $context) {
         'id' => $params['contactId'],
```

Comment:
Any reason for the change in the permission of this file?

Reply:
None at all. Set to 644 again.